apps/webapp/app/routes/_app.orgs.$organizationSlug.projects.v3.$projectParam.apikeys/route.tsx

```diff
@@ -85,14 +85,12 @@ export default function Page() {
         <div className="flex flex-col">
           <Table containerClassName="border-t-0">
             <TableHeader>
-              <TableRow>
-                <TableHeaderCell>Environment</TableHeaderCell>
-                <TableHeaderCell>Secret key</TableHeaderCell>
-                <TableHeaderCell>Key generated</TableHeaderCell>
-                <TableHeaderCell>Latest version</TableHeaderCell>
-                <TableHeaderCell>Env vars</TableHeaderCell>
-                <TableHeaderCell hiddenLabel>Actions</TableHeaderCell>
-              </TableRow>
+              <TableHeaderCell>Environment</TableHeaderCell>
+              <TableHeaderCell>Secret key</TableHeaderCell>
+              <TableHeaderCell>Key generated</TableHeaderCell>
+              <TableHeaderCell>Latest version</TableHeaderCell>
+              <TableHeaderCell>Env vars</TableHeaderCell>
+              <TableHeaderCell hiddenLabel>Actions</TableHeaderCell>
```

Comment:
"You must place `<tr>` elements inside `<thead>` to contain your `<th>` cells. The correct structure is: `<thead><tr><th>Header Cell</th></tr></thead>`. [1](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/thead) [2](https://www.w3.org/TR/html401/struct/tables.html)"